jobs/build/release-schedules/Jenkinsfile

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env groovy
+
+node {
+    try {
+        checkout scm
+        commonlib = load("pipeline-scripts/commonlib.groovy")
+        slacklib = commonlib.slacklib
+        
+
+        stage("install the schedule-generator tool") {
+            def install = [
+                "rm -rf ci-tools",
+                "git clone https://github.com/openshift/ci-tools.git",
+                "cd ci-tools",
+                "go install ./cmd/branchingconfigmanagers/schedule-generator/...",
+                "cd .."
+            ]
+
+            cmd = (install).flatten().join('\n')
+
+            echo "shell cmd: ${cmd}"
+            out = commonlib.shell(
+                returnStdout: true,
+                script: cmd
+            )
+            
+            echo "out: ${out}"
+        }
+
+        sshagent(["openshift-bot"]) {
+            stage("clone the ocp release schedules") {
+                def clone = [
+                    "rm -rf schedule",
+                    "git clone --single-branch --branch master git@gitlab.cee.redhat.com:ocp-release-schedule/schedule.git",
+                    "cd schedule"
+                ]
+                cmd = (clone).flatten().join('\n')
+
+                echo "shell cmd: ${cmd}"
+                out = commonlib.shell(
+                    returnStdout: true,
+                    script: cmd
+                )
+                
+                echo "out: ${out}"
+            }
+        }
+
+
+         stage("generate the ocp release schedules") {
+            def schedules_cmd = "schedule-generator --kubeconfig /home/jenkins/kubeconfigs/sa.release-schedules-cm-updater.app.ci.config --schedule-dir . --dry-run=false"
```

Comment:
@jupierce How can we maintain the kubeconfig file? Is there any private repo that holds all those files?

Reply:
Same security constraints here. I'll need to install the kubeconfig manually.